sw/device/silicon_creator/mask_rom/romextimage_ptrs.h

```diff
@@ -0,0 +1,52 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_MANIFEST_PTR_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_MANIFEST_PTR_H_
+
+#include "sw/device/silicon_creator/lib/manifest.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+static_assert((TOP_EARLGREY_EFLASH_SIZE_BYTES % 2) == 0,
+              "Flash size is not divisible by 2");
+
+#ifndef OT_OFF_TARGET_TEST
+/**
+ * Returns a pointer to the manifest of the ROM_EXT image stored in flash
+ * slot A.
+ *
+ * @return Pointer to the manifest of the ROM_EXT image in slot A.
+ */
+inline const manifest_t *romextimage_slot_a_manifest_ptr_get(void) {
+  return (const manifest_t *)TOP_EARLGREY_EFLASH_BASE_ADDR;
+}
```

Comment:
Is there value in having these in a separate header file or would it be more straightfoward if these were part of `manifest.h`?

Reply:
Mostly to keep `manifest.h` limited to the manifest struct & related helpers (assuming it will be reused in ROM_EXT and BL0 and they won't need to access previous stages' manifests) and to keep these raw getters somewhat private, i.e. these should not be used instead of `romextimage_manifest_get`.